### PR TITLE
updated the instance types to be c3.2xlarge and adjusted the price

### PR DESCRIPTION
**Pull request by grenade**
Amy requested that I merge these changes from her fork. Some context at: https://bugzilla.mozilla.org/show_bug.cgi?id=1166448

### CODE REVIEW

**Comment by grenade** [comment]
Anyone know what the [travis build failure](https://travis-ci.org/mozilla/build-cloud-tools/builds/71922371#L357) means?

File: configs/watch_pending.cfg [hunk 1]
```diff
@@ -16,15 +16,21 @@
...
         "^horizon_(?!try)\\S+_linux(32|64)(-debug)?": "bld-linux64",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)?(.*(?! try)).* (pgo-)?build": "b-2008",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)?.* nightly": "b-2008",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)? try": "try-2008",
         "^b2g_try_linux(32|64)_gecko(-debug)?": "try-linux64",
         "^Ubuntu VM 12.04 (?!x64).*": "tst-linux32",
         "^Ubuntu (Code Coverage )?VM 12.04 x64.*": "tst-linux64",
         "^Ubuntu Mulet VM 12.04 x64.*": "tst-linux64",
         "^Ubuntu ASAN VM 12.04 x64.*": "tst-linux64",
-        "^Ubuntu TSAN VM 12.04 x64.*": "tst-linux64",
```
Comment (grenade) [inline configs/watch_pending.cfg]: reverted removal (arr: I don't think there were any matches, but it doesn't hurt to leave it in)

**Comment by amyrrich** [comment]
To be clear, I haven't tested the changes for slavealloc.py and watch_pending.cfg, just asking Rob to pick up where I left off on adding that functionality in since he's getting more involved with the AWS stuff now. Some testing is definitely in order. :}

**Comment by rail** [comment]
:+1:

**Pull request merged**